Gum/Commands/EditCommands.cs

```diff
@@ -465,6 +466,81 @@ public void DuplicateSelectedElement()
 
         }
 
+        public void CreateComponent()
```

Comment:
This method doesn't always create the component - it depends on error checking and if the user clicks OK. I would suggest naming this "ShowCreateComponentFromInstancesDialog" or something to that effect to indicate that the dialog is being shown but that the component may not actually be created.